web/src/app/success-password-1st-login/page.tsx

```diff
@@ -0,0 +1,37 @@
+import { Button } from "@/components/ui/button";
+import Link from "next/link";
+import SuccessImage from "./assets/success.svg";
+
+export default async function SuccessPasswordFirstLogin() {
+    return (
+        <section className="mt-48">
+            <div className="flex items-center justify-center ">
+                <SuccessImage />
+            </div>
+
+            <div>
+                <p className="flex flex col gap-y-12 p-12 items-center justify-center font-sans font-extrabold text-3xl text-indigo-950">
+                    Tudo certinho!
+                </p>
+            </div>
+
+            <div>
+                <p className="flex flex col gap-6 items-center justify-center text-base font-bold text-indigo-950">
+                    Pronto para começar a agendar?
+                </p>
+
+                <div className="flex flex-col p-6 gap-2">
+                    <Button className="font-bold text-base">
+                        <Link href="">VAMOS LÁ!</Link>
```

Comment:
```suggestion
                        {/* TODO: adicionar caminho quando a aplicação estiver fazendo agendamentos */}
                        <Link href="">VAMOS LÁ!</Link>
```

Só adicionar esse TODO pra garantir que não vamos deixar passar esse link sem adicionar nada ao caminho.